app/ide-desktop/lib/client/src/bin/server.ts

```diff
@@ -85,32 +88,41 @@ export class Server {
                     http: this.config.port,
                     handler: this.process.bind(this),
                 },
-                (err: any) => {
+                err => {
                     if (err) {
-                        logger.error(`Error creating server:`, err.http)
+                        contentConfig.LOGGER.error(`Error creating server:`, err.http)
                         reject(err)
                     }
-                    logger.log(`Server started on port ${this.config.port}.`)
-                    logger.log(`Serving files from '${process.cwd()}/${this.config.dir}'.`)
+                    contentConfig.LOGGER.log(`Server started on port ${this.config.port}.`)
+                    contentConfig.LOGGER.log(
+                        `Serving files from '${path.join(process.cwd(), this.config.dir)}'.`
+                    )
                     resolve()
                 }
             )
         })
     }
 
-    process(request: { url: string }, response: any) {
-        const url = request.url.split('?')[0]
-        const resource = url == '/' ? '/index.html' : request.url
-        let resource_file = `${this.config.dir}${resource}`
-        fs.readFile(resource_file, (err: any, data: any) => {
+    process(request: http.IncomingMessage, response: http.ServerResponse) {
+        const requestUrl = request.url
+        if (requestUrl === undefined) {
+            contentConfig.LOGGER.error('Request URL is null.')
+            return
+        }
+        const url = requestUrl.split('?')[0]
+        const resource = url === '/' ? '/index.html' : requestUrl
+        const resourceFile = `${this.config.dir}${resource}`
+        fs.readFile(resourceFile, (err, data) => {
             if (err) {
-                logger.error(`Resource '${resource}' not found.`)
+                contentConfig.LOGGER.error(`Resource '${resource}' not found.`)
```

Comment:
TBH, I much more preferred the older `logger.error` over `contentConfig.LOGGER` syntax. The new one is super verbose. Can we keep the old one, please? If you prefer the new one, could you explain why, please?

Reply:
this was just auto-refactored because we were importing as `{ logger }` before. i can extract it to a const so it will be `LOGGER.error` - would that work?

Reply:
As I've written below, I think it should be `logger.error`, as we are not writing `CONSOLE.log`. Do you agree?

Reply:
yup - it's already been changed to `logger.error`